### PR TITLE
First pass of changes based on Dry Run feedback

### PR DESCRIPTION
**Pull request by renvrant**
- Adds not in this talk and project dependencies back to the intro
- Updates section about adding your reducer with better explanation
- Adds some diagrams
- Adds some deliberately silly diagrams
- Removes FP section in validation
- Simplifies some of the validation examples
- Breaks the conclusion points out into multiple slides
- Updates slide styles

Possibly more to come...

@danielfigueiredo

### CODE REVIEW

File: src/content/store-and-form-setup.md [hunk 1]
```diff
@@ -26,27 +38,46 @@ export interface IBioSummary {
...
 ```
 
 ---
 
 ## Create form reducer
-Including a path in the payload makes this reducer reusable for other forms
+Our form changes will be merged at the provided path
 
 ```ts
 import { lensPath, assocPath, merge } from 'ramda';
```
Comment (renvrant) [inline src/content/store-and-form-setup.md:80]: @danielfigueiredo You mentioned we could tidy this example up and use only `path` from Ramda?

File: src/content/conclusion.md [hunk 2]
```diff
@@ -2,15 +2,23 @@
...
+## Why should I bother?
 
 ---
 
-## Why bother?
-- On large-scale projects, the benefit will outweigh the cost
-- A single reducer can be used for many forms
-- Generic actions and validators can be re-used
-- Set it up once, use it many times (even in other frameworks like NativeScript!)
+## It's re-usable
```
Comment (danielfigueiredo) [inline src/content/conclusion.md:9]: I like that, single slides with just small sentences which we can expand on

File: src/content/images/store-component-flow.svg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (danielfigueiredo) [inline src/content/images/store-component-flow.svg:1]: OMG I was wordless with the wizard + dragon thing
that is amazing!
so for this one store-component-flow-lolz.png you're just missing the selectors on top of the wizard's head
between store and component, when it's coming back to the component

Comment (danielfigueiredo) [reply src/content/images/store-component-flow.svg:1]: `store-component-flow.png` should also have selectors on the left between store and components

File: src/content/introduction.md [hunk 4]
```diff
@@ -33,7 +33,19 @@
...
+- Your forms + Redux magic
 - Adding form data to state
 - Supporting multi-entry fields
 - Framework-agnostic validation with selectors
+
+---
+
+## Not in this talk
+- Redux 101
+
```
Comment (danielfigueiredo) [inline src/content/introduction.md:45]: maybe add here:
- Functional Programming
- Reselect
- angular-redux/store
- Ramda

we could use all these bullets to briefly describe what each of these libraries does when we're talking

File: src/content/store-and-form-setup.md [hunk 5]
```diff
@@ -2,11 +2,23 @@
 
 ---
 
-## Plan the shape of your form 
-You can structure your store however you like
-`form.character` will represent a character form in this example
+## One reducer to rule them all
```
Comment (danielfigueiredo) [inline src/content/store-and-form-setup.md:5]: that is amazing, we're just missing a ring in here! ahahaha

File: src/content/introduction.md [hunk 6]
```diff
@@ -12,7 +12,7 @@
 
 ---
 
-## Redux can power up your forms
+## Redux can level up your forms
```
Comment (danielfigueiredo) [inline src/content/introduction.md:15]: Redux can bring you to an epic level!

File: src/content/store-and-form-setup.md [hunk 7]
```diff
@@ -26,27 +38,46 @@ export interface IBioSummary {
...
+export interface IEquipment {
+  weaponName?: string;
+  weaponType: string;
+  armorType: string;
+}
+```
+
+---
+
 ## Setting up actions
```
Comment (danielfigueiredo) [inline src/content/store-and-form-setup.md:60]: What do you think about something like:
`Setting up actions (you don't wanna be surprised)`
or
`Setting up your spells, ops.. actions!`

File: src/content/store-and-form-setup.md (hunk 1 above)
Comment (danielfigueiredo) [reply src/content/store-and-form-setup.md:80]: `lensPath + view = path`
we could just `path(action.payload.path),`
and `path` comes from `import { path } from 'ramda'`
I can help out with that

File: src/content/store-and-form-setup.md [hunk 8]
```diff
@@ -89,70 +142,31 @@ export class CharacterForm {
...
-  <input
-    type="number"
-    name="age"
-    [(ngModel)]="characterForm.bioSummary.age">
-</form>
-```
-
----
-
 ## Listen and dispatch
```
Comment (danielfigueiredo) [inline src/content/store-and-form-setup.md:145]: maybe some joke with `Reactions and casting`

File: src/content/validation.md [hunk 9]
```diff
@@ -70,101 +70,47 @@ After selecting, we can use `isFormValid$` with the async pipe in our template
...
-
-const ageValidationSelector = createSelector(
+const isAgeValid = createSelector(
   bioSummarySelector,
   (bioSummary: IBioSummary) => {
     switch (bioSummary.race) {
-      case 'Human': return humanAgeValid;
-      case 'Elf': return elfAgeValid;
-      case 'Tiefling': return tieflingAgeValid;
+      case 'Human': return bioSummary.age < 65;
```
Comment (danielfigueiredo) [inline src/content/validation.md:77]: I will probably remove this whole example

**Review by danielfigueiredo — COMMENTED** [review]
Overall looks great!

File: src/content/introduction.md (hunk 4 above)
Comment (renvrant) [reply src/content/introduction.md:45]: I feel like we mention most of these things in the project dependencies but we also explain them as much as they're needed... Redux is the only thing we don't really give an overview of. Then again, maybe we should give an overview?

File: src/content/store-and-form-setup.md (hunk 7 above)
Comment (renvrant) [reply src/content/store-and-form-setup.md:60]: Hmm, I'm trying to think of something that would work here but not be too long or confusing 🤔 I don't quite get the first one with `you don't wanna be surprised`, though... I wonder if we can do a pun with combat actions? or maybe somewhere else something about rolling for initiative.

File: src/content/store-and-form-setup.md (hunk 8 above)
Comment (renvrant) [reply src/content/store-and-form-setup.md:145]: Maybe like... `Make the magic happen`?

**Pull request merged**